ngraph/test/type_prop/convolution_backprop_data.cpp

```diff
@@ -0,0 +1,1113 @@
+//*****************************************************************************
```

Comment:
Missing type prop tests with setting `output_padding `attribute for v0 and v1.

Reply:
Done